### DISCUSSION TITLE
GAMES103 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#125] [Announcements]
# GAMES103 - CS自学指南

CS自学指南

https://csdiy.wiki/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9B%BE%E5%BD%A2%E5%AD%A6/GAMES103/

**Comment by ericJay4arch** [comment]
新创建了一个QQ群，群号：785869800。本人学完了101与103，正在做物理模拟的paper复现，苦于老师没有太多指导，希望学习图形学的同学能一起交流，一起讨论。

**Comment by rijuyuezhu** [comment]
请问有推荐的大学物理的课程推荐的吗? 谢谢

**Reply by 520Enterprise** [reply]
南京大学肖明文老师的《大学物理》很不错(逃)

**Comment by linearalgebrayhz** [comment]
寒假把103学了，103涵盖的几大板块：1. 刚体模拟 2. 质点弹簧系统（布料）3. 弹性体有限元 4. 流体模拟 之间依赖性不会特别强，分别学也是完全可以的。王老师是做布料模拟的大佬，他上课真的讲得很清楚对于一些技术的实现动机有很棒的解释。个人觉得这门课在上过一点数值计算和优化相关的知识之后来看会比较合适，因为课上很多数值计算方面的工具基本上会拿来直接用而不会有太多的解释。此外这门课的作业环境也比较友好，直接用Unity，写一些C#。Lab比较推荐完成bonus task加深理解。
